Buildings/Controls/OBC/Utilities/PIDWithInputGains.mo

```diff
@@ -0,0 +1,749 @@
+          fillColor={255,255,255},
+          fillPattern=FillPattern.Solid),
+        Text(
+          visible=(controllerType == Buildings.Controls.OBC.CDL.Types.SimpleController.P),
+          extent={{-32,-22},{68,-62}},
+          lineColor={0,0,0},
+          textString="P",
+          fillPattern=FillPattern.Solid,
+          fillColor={175,175,175}),
+        Text(
+          visible=(controllerType == Buildings.Controls.OBC.CDL.Types.SimpleController.PI),
+          extent={{-26,-22},{74,-62}},
+          lineColor={0,0,0},
+          textString="PI",
+          fillPattern=FillPattern.Solid,
+          fillColor={175,175,175}),
+        Text(
+          visible=(controllerType == Buildings.Controls.OBC.CDL.Types.SimpleController.PD),
+          extent={{-16,-22},{88,-62}},
+          lineColor={0,0,0},
+          fillPattern=FillPattern.Solid,
+          fillColor={175,175,175},
+          textString="P D"),
+        Text(
+          visible=(controllerType == Buildings.Controls.OBC.CDL.Types.SimpleController.PID),
+          extent={{-14,-22},{86,-62}},
+          lineColor={0,0,0},
+          textString="PID",
+          fillPattern=FillPattern.Solid,
+          fillColor={175,175,175}),
+        Polygon(
+          points={{-80,82},{-88,60},{-72,60},{-80,82}},
+          lineColor={192,192,192},
+          fillColor={192,192,192},
+          fillPattern=FillPattern.Solid),
+        Line(
+          points={{-80,68},{-80,-100}},
+          color={192,192,192}),
+        Line(
+          points={{-90,-80},{70,-80}},
+          color={192,192,192}),
+        Polygon(
+          points={{74,-80},{52,-72},{52,-88},{74,-80}},
+          lineColor={192,192,192},
+          fillColor={192,192,192},
+          fillPattern=FillPattern.Solid),
+        Text(
+          extent={{-150,150},{150,110}},
+          textString="%name",
+          textColor={0,0,255}),
+        Line(
+          points={{-80,-80},{-80,-22}},
+          color={0,0,0}),
+        Line(
+          points={{-80,-22},{6,56}},
+          color={0,0,0}),
+        Line(
+          points={{6,56},{68,56}},
+          color={0,0,0}),
+        Rectangle(
+          extent=DynamicSelect({{100,-100},{84,-100}},{{100,-100},{84,-100+(y-yMin)/(yMax-yMin)*200}}),
+          fillColor={175,175,175},
+          fillPattern=FillPattern.Solid,
+          pattern=LinePattern.None,
+          lineColor={0,0,0})}),
+    Diagram(
+      coordinateSystem(
+        extent={{-220,-200},{220,200}}), graphics={Rectangle(
+          extent={{-56,180},{-24,-16}},
+          fillColor={215,215,215},
+          fillPattern=FillPattern.Solid,
+          pattern=LinePattern.None), Text(
+          extent={{-52,184},{-28,156}},
+          pattern=LinePattern.None,
+          fillColor={215,215,215},
+          fillPattern=FillPattern.Solid,
+          lineColor={0,0,0},
+          textString="PID")}),
+    Documentation(
+      info="<html>
+<p>
+PID controller in the standard form
```

Comment:
Good to explain what the block is, but please also add how this block differs from `CDL.Continuous.PIDWithReset` as this is not obvious to users.

Reply:
Thanks, I have highlighted the difference between this block and <code>CDL.Continuous.PIDWithReset</code>